src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -268,16 +268,41 @@ public String getTypeDescription() {
       help = "Whether to upload locally executed action results to the remote cache.")
   public boolean remoteUploadLocalResults;
 
+  @Deprecated
   @Option(
       name = "incompatible_remote_build_event_upload_respect_no_cache",
       defaultValue = "false",
       documentationCategory = OptionDocumentationCategory.REMOTE,
       effectTags = {OptionEffectTag.UNKNOWN},
+      deprecationWarning =
+          "--incompatible_remote_build_event_upload_respect_no_cache has been deprecated in favor"
+              + " of --experimental_remote_build_event_upload.",
```

Comment:
So does `--experimental_remote_build_event_upload=minimal` also imply `--incompatible_remote_build_event_upload_respect_no_cache`? This deprecation should probably mention the `minimal` value?

Reply:
> So does `--experimental_remote_build_event_upload=minimal` also imply `--incompatible_remote_build_event_upload_respect_no_cache`?

No. I think with `--experimental_remote_build_event_upload=minimal`, there is no need for `incompatible_remote_build_event_upload_respect_no_cache`, right?

Updated message to mention `minimal`.